src/ClassFileLocator.php

```diff
@@ -122,6 +122,17 @@ public function accept()
                     if ($i > 0 && is_array($tokens[$i - 1]) && $tokens[$i - 1][0] === T_DOUBLE_COLON) {
                         break;
                     }
+
+                    // ignore anonymous classes on PHP 7.1 and greater
+                    if (&& $i >= 2
```

Comment:
Um... `&&` to start the conditional...? Also, will `$i >= 2` only ever happen on PHP 7.1, or will this work on any version?